samples/xmlserializergenerator-instructions.md

```diff
@@ -7,7 +7,7 @@ You can start using the tool today following the instructions below.
 ## Prerequisites
 
```

Comment:
Can we add a note something like below?

"The following is required for Microsoft.XmlSerializer.Generator to work. You can use command `dotnet --info` to check which versions of .NET Core SDK and runtime you may already have installed."

Reply:
Added the above note.